src/pages/AddData.tsx

```diff
@@ -31,6 +32,7 @@ interface BulkUploaderProps {
 	onBulkUpload: () => void;
 }
 const BulkUploader = ({ onBulkUpload }: BulkUploaderProps) => {
+	const [systemSource] = useSystemSource();
 	const createPresignedPost = usePresignedPostCallback();
 	const registerBulkUpload = useRegisterBulkUploadCallback();
 	const handleUpload = async (file: File) => {
```

Comment:
I believe this should be using `useCallback`, because right now it appears to be recreating this callback on every render of the `<BulkUploader>` component.

However, this is not a new problem, so it doesn't need to be fixed in this PR.